src/Game/zLight.cpp

```diff
@@ -15,22 +20,82 @@
 #pragma GLOBAL_ASM("asm/Game/zLight.s", "zLightResolveLinks__Fv")
 
 // func_8009E330
-#pragma GLOBAL_ASM("asm/Game/zLight.s", "zLightDestroyAll__Fv")
+// #pragma GLOBAL_ASM("asm/Game/zLight.s", "zLightDestroyAll__Fv")
```

Comment:
remove the pragma lines instead of commenting them out if the function is decompiled fully